src/PanelContentsWithSubpanel/PanelContentsWithSubpanel.tsx

```diff
@@ -5,16 +5,16 @@ import { useCloseSubpanelWhenParentPanelCloses } from "./useCloseSubpanelWhenPar
 
 export const PanelContentsWithSubpanel = ({
   isLeftPanelOpen,
```

Comment:
Im making this interface more permissive, to facilitate less complexity in the calling code